### ISSUE TITLE
Report here about false detections or undetected hack 

### ISSUE THREAD

**Issue by 2020karaulov2020** [2020karaulov2020/UnrealDemoScanner#7]
**Report here about false detections or undetected hack**

**Comment by Shireku** [comment]
Ложные детекты на Ideal Jump?
[демка](https://transfiles.ru/cewx6)
![Screenshot_1](https://user-images.githubusercontent.com/24224043/91002521-3e76e300-e612-11ea-8e7d-5d4bc5d863cc.jpg)

**Comment by 2020karaulov2020** [comment]
> Ложные детекты на Ideal Jump?
> [демка](https://transfiles.ru/cewx6)
> ![Screenshot_1](https://user-images.githubusercontent.com/24224043/91002521-3e76e300-e612-11ea-8e7d-5d4bc5d863cc.jpg)

Fixed in https://github.com/2020karaulov2020/UnrealDemoScanner/commit/2a8489bdb8a544de7fb3282c213688bb5c470763

**Comment by Shireku** [comment]
![Screenshot_1](https://user-images.githubusercontent.com/24224043/91709319-ecabfb00-ebc5-11ea-9302-03e68ac1d140.jpg)
Демка не сканится и появляются ошибки...это проблема с демкой?
[Демка](https://yadi.sk/d/kNjprO6SvauOtA)

**Comment by 2e45846a9cfe** [comment]
False detection [RETURN TO GAME FEATURE]
 in demo : 
[womenpower_3_1(RETURN TO GAME FALSE).zip](https://github.com/2020karaulov2020/UnrealDemoScanner/files/5212370/womenpower_3_1.RETURN.TO.GAME.FALSE.zip)

**Comment by 2020karaulov2020** [comment]
_(BHOP TYPE 2 FALSE )_

**Comment by BorisKotov174Rus** [comment]
доброй ночи интересует такие вот сообщения
![8WxuEppiYYE](https://user-images.githubusercontent.com/77694405/105081379-ee625680-5ab3-11eb-90c7-0e21aa64cf51.jpg)
![nhHPazCO0i8](https://user-images.githubusercontent.com/77694405/105081384-ef938380-5ab3-11eb-901a-e8cf3079fa3e.jpg)

**Comment by 2020karaulov2020** [comment]
> доброй ночи интересует такие вот сообщения
> ![8WxuEppiYYE](https://user-images.githubusercontent.com/77694405/105081379-ee625680-5ab3-11eb-90c7-0e21aa64cf51.jpg)
> ![nhHPazCO0i8](https://user-images.githubusercontent.com/77694405/105081384-ef938380-5ab3-11eb-901a-e8cf3079fa3e.jpg)

Меня тоже, прикрепите демо в архиве (zip).

**Comment by BorisKotov174Rus** [comment]
[m1.zip](https://github.com/2020karaulov2020/UnrealDemoScanner/files/5845783/m1.zip)
 демо игрока выдает так же ошибки что и оратора выше постом

**Comment by 2020karaulov2020** [comment]
> [m1.zip](https://github.com/2020karaulov2020/UnrealDemoScanner/files/5845783/m1.zip)
> демо игрока

Все нормально это не POV демо а HLTV.

**Comment by BorisKotov174Rus** [comment]
спасибо.

**Comment by Andrey111q** [comment]
Добрый вечер, постоянно показывает фальш аим, подскажите что может быть?
![image](https://user-images.githubusercontent.com/77932306/105637366-04469180-5e8f-11eb-948e-7ab5f9cf3229.png)
![image](https://user-images.githubusercontent.com/77932306/105637376-0c063600-5e8f-11eb-8848-83132a51d798.png)

**Comment by q3wzeck** [comment]
![image](https://user-images.githubusercontent.com/15782415/125363070-a5130580-e378-11eb-8d9b-fc03c813b2d4.png)
https://drive.google.com/file/d/1UTuOqju-8HUlD2q8WD4qy8m4buGnrN-I/view?usp=sharing

**Comment by THORMENTY** [comment]
![image](https://user-images.githubusercontent.com/87832756/126707206-a1b6bd82-2b2d-4617-872b-112926be18f8.png)
![image](https://user-images.githubusercontent.com/87832756/126707220-2c86065e-280c-4a2c-8846-21c20646d678.png)

**Comment by 2020karaulov2020** [comment]
@THORMENTY you can share demo

**Comment by THORMENTY** [comment]
> @THORMENTY você pode compartilhar uma demonstração

https://drive.google.com/file/d/14a4IX-4NLUGZT69xgRNFwQJN8ilc8crT/view?usp=sharing

**Comment by 2020karaulov2020** [comment]
> > @THORMENTY você pode compartilhar uma demonstração
> 
> https://drive.google.com/file/d/14a4IX-4NLUGZT69xgRNFwQJN8ilc8crT/view?usp=sharing

@THORMENTY this is not false detections. This is warns. Player has very bad internet connection and loss 20-30 packets in seconds. (judging by the data from the demo file) 
(LAG) - warn, but player has loss packets at this time, and 99% that this detection is false. If these lags not caused by hack.
